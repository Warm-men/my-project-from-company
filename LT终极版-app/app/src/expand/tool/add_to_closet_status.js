const Column = {
  NewArrivalCollection: 'NewArrivalCollection', //会员上新
  RecentHotCollection: 'RecentHotCollection', //非会员首页热门单品
  OccasionCollection: 'OccasionCollection', //首页场合
  Collection: 'Collection', //专题
  Brand: 'Brand', //品牌详情列表
  Clothing: 'Clothing', //选衣-衣服
  Accessory: 'Accessory', //选衣-饰品
  Closet: 'Closet', //愿望衣橱
  SatisfiedCloset: 'SatisfiedCloset', //愿望衣橱进满分单品
  SwapSatisfiedCloset: 'SwapSatisfiedCloset', //换装进满分单品
  Recommend: 'Recommend', //会员首页为你推荐
  SimilarProducts: 'SimilarProducts', //商品详情页相似单品
  ClothingOccasion: 'ClothingOccasion', //选衣-衣服场合
  AccessoryOccasion: 'AccessoryOccasion', //选衣-饰品场合
  ProductsOccasion: 'ProductsOccasion', //
  Outfits: 'Outfits', //推荐搭配
  LookBooks: 'LookBooks', //单个LookBook专题详情
  Lookbook: 'Lookbook', //单个Look
  ToteSwapCollectionCity: 'ToteSwapCollectionCity',
  ToteSwapCollectionConstellation: 'ToteSwapCollectionConstellation',
  ToteSwapCollectionAge: 'ToteSwapCollectionAge',
  ToteSwapCollectionOccupation: 'ToteSwapCollectionOccupation',
  ToteSwapCollectionShape: 'ToteSwapCollectionShape',
  ToteSwapCollectionNextWeek: 'ToteSwapCollectionNextWeek',
  ToteSwapCollectionRecommend: 'ToteSwapCollectionRecommend',
  ToteSwapClothing: 'ToteSwapClothing', //D
  ToteSwapAccessory: 'ToteSwapAccessory', //D
  ToteSwapCloset: 'ToteSwapCloset', //换装的愿望衣橱
  LatestCustomerPhotos: 'LatestCustomerPhotos', //D
  MyCustomerPhotos: 'MyCustomerPhotos', //我的-晒单
  CustomerPhotoDetails: 'CustomerPhotoDetails', //晒单详情页
  CustomerPhotoFinished: 'CustomerPhotoFinished', //我的历史衣箱内完成的-查看晒单
  CustomerPhotosReviewed: 'CustomerPhotosReviewed', //晒单评价列表
  ProductCustomerPhoto: 'ProductCustomerPhoto', //商品详情页查看晒单
  CurrentTote: 'CurrentTote', //在手衣箱
  PastTote: 'PastTote', //历史衣箱
  ShoppingCar: 'ShoppingCar', //购物车
  PaymentPending: 'PaymentPending', //待付款
  Link: 'Link', //网页
  ClothingProduct: 'ClothingProduct', //新选衣-衣服
  ClothingCustomerPhoto: 'ClothingCustomerPhoto', //新选衣-晒单
  ClothingCollection: 'ClothingCollection', //新选衣-专题
  ClothingLookBook: 'ClothingLookBook', //新选衣-Look
  ProductLookBook: 'ProductLookBook', //商品详情页Looks
  ProductLooks: 'ProductLooks', //进入 新选衣-Look
  SearchProducts: 'SearchProducts' //搜索产品页
}

const dataSources = {
  // QUERY_SWAP_COLLECTION,//推荐
  // QUERY_BANNER_GROUP,//最新上新
  // QUERY_BRAND_DETAIL,//品牌详情
}

export { Column, dataSources }
