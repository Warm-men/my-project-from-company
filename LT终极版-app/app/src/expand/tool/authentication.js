import dateFns from 'date-fns'

/**
 * 用户权限
 * @param {*obj} customer
 */
const authentication = customer => {
  return {
    isValidSubscriber: isValidSubscriber(customer),
    isExpiredSubscriber: isExpiredSubscriber(customer),
    isSubscriber: isSubscriber(customer),
    expiresInDays: expiresInDays(customer),
    subscriptionDate: subscriptionDate(customer)
  }
}

/**
 * 订阅天数
 * @param {*obj} customer
 */
const differenceDays = customer => {
  const { subscription } = customer
  if (!subscription) return 1
  const currentTime = dateFns.format(new Date(), 'YYYY-MM-DD')
  const subscriptionTime = dateFns.format(
    subscription.billing_date,
    'YYYY-MM-DD'
  )
  return dateFns.differenceInDays(currentTime, subscriptionTime)
}

/**
 * 截止日期
 * @param {*obj} customer
 */
const subscriptionDate = customer => {
  const { subscription } = customer
  if (!subscription) return ''
  if (!subscription.billing_date) return ''
  const subscriptionTime = dateFns.format(
    subscription.billing_date,
    'YYYY年MM月DD日'
  )
  return subscriptionTime
}

/**
 * 是否订阅 true or false
 * @param {*obj} customer
 */
const isValidSubscriber = customer => differenceDays(customer) <= 0

/**
 * 是否过期 true or false
 * @param {*obj} customer
 */
const isExpiredSubscriber = customer => {
  const billing_date =
    customer.subscription && customer.subscription.billing_date
  return (
    isSubscriber(customer) &&
    (billing_date === 'N/A' ||
      billing_date === null ||
      differenceDays(customer) > 0)
  )
}

/**
 * 是否订阅
 * @param {*obj} customer
 */
const isSubscriber = customer =>
  customer.subscription && customer.subscription.id ? true : false

/**
 * 过期天数
 * @param {*obj} customer
 */
const expiresInDays = customer =>
  differenceDays(customer) > 0 ? differenceDays(customer) : -1

export default authentication
