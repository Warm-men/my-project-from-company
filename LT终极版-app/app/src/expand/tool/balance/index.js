import Stores from '../../../stores/stores'
import { getBalance } from '../../../request'

/**
 * 刷新信用账户金额
 */

const updateBalance = async balance => {
  let response
  if (balance === undefined || balance === null) {
    response = await getBalance()
    balance = response.data.me.credit_account.balance
  }
  Stores.currentCustomerStore.updateBalance(balance)
}
export { updateBalance }
