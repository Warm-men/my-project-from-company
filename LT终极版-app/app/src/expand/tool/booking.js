/**
 * 预约归还时间
 */
import dateFns from 'date-fns'

const booking = (deadline, hour, minute) => {
  return {
    bookingDate: date(deadline, hour, minute)
  }
}

// NOTE：后台没有的时间，都采用17:00
const defaultDeadLine = '17:00'

const date = (deadline, hour, minute) => {
  //add hour, minute just for test
  let getHours = hour ? hour : new Date().getHours() //小时
  let getMinutes = minute ? minute : new Date().getMinutes() //分钟
  let timeData = []
  const minutes = ['00分', '30分']
  // NOTE:有传入使用传入截止日期，没有使用默认的
  const deadlineList = deadline
    ? deadline.split(':')
    : defaultDeadLine.split(':')

  // NOTE:截止的时间
  const lastTime = parseInt(deadlineList[0], 10)
  const lastMoment = parseInt(deadlineList[1], 10)

  // NOTE:最后下单时间的分钟list
  const lastMomentList = lastMoment === 30 ? minutes : ['00分']

  for (let i = 0; i < 2; i++) {
    let hours = [] //今天剩下的小时组合

    let days = {}
    if (i === 0 && getHours + 1 >= lastTime && getMinutes >= lastMoment) {
      timeData = []
    } else {
      const hoursStartAt = i === 1 || getHours < 9 ? 9 : getHours + 1
      for (let h = hoursStartAt; h <= lastTime; h++) {
        let hoursObject = {}
        let hoursName = `${h}时`
        if (h === lastTime) {
          hoursObject[hoursName] = ['00分']
        } else if (h === getHours + 1) {
          if (getMinutes < 30 && getMinutes > 0) {
            hoursObject[hoursName] = ['30分']
          } else {
            hoursName = `${h + 1}时`
            if (h + 1 == lastTime) {
              hoursObject[hoursName] = ['00分']
            } else {
              hoursObject[hoursName] = minutes
            }
            h++
          }
        } else {
          hoursObject[hoursName] = h < lastTime ? minutes : lastMomentList
        }
        hours.push(hoursObject)
      }

      let dayName = dateFns.format(dateFns.addDays(new Date(), i), 'MM月DD日')

      days[dayName] = hours

      timeData.push(days)

      hours = []
    }
  }
  return timeData
}

export default booking
