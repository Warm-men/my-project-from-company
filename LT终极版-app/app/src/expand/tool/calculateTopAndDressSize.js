const cupSize = {
  A: 10,
  B: 12.5,
  C: 15,
  D: 17.5,
  E: 20
}

// const DRESS_SIZES = [
//   { name: 'XS', type: 2 },
//   { name: 'S', type: 4 },
//   { name: 'M', type: 8 },
//   { name: 'L', type: 12 },
//   { name: 'XL', type: 14 }
// ]
// const TOP_SIZES_ABBR = [
//   { name: 'XS', type: 'X-Small' },
//   { name: 'S', type: 'Small' },
//   { name: 'M', type: 'Medium' },
//   { name: 'L', type: 'Large' },
//   { name: 'XL', type: 'X-Large' }
// ]

const calculateSize = (bra_size, cup_size) => {
  const size = parseInt(bra_size + cupSize[cup_size], 10)
  if (size < 93) {
    return {
      top_size: 'Small',
      dress_size: 4
    }
  } else if (size <= 97) {
    return {
      top_size: 'Medium',
      dress_size: 8
    }
  } else if (size <= 105) {
    return {
      top_size: 'Large',
      dress_size: 12
    }
  } else {
    return {
      top_size: 'X-Large',
      dress_size: 14
    }
  }
}

export default calculateSize
