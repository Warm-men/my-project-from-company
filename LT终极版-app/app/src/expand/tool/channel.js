import { Platform } from 'react-native'
import RNAnalytics from 'react-native-letote-baidumjt'

const channel =
  Platform.OS === 'ios' ? 'AppStore' : RNAnalytics.getChannel() || 'letote'

export default channel
