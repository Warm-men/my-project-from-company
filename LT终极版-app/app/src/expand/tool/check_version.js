import React from 'react'
import { Platform } from 'react-native'
import Stories from '../../stores/stores'
import { GET, SERVICE_TYPES } from '../services/services'
import dateFns from 'date-fns'
import OccasionBanner from '../../../storybook/stories/alert/occasion_banner'

const getNewestVersionALert = () => {
  const { appStore } = Stories
  if (!appStore.versionData) {
    return
  }
  let data
  if (Platform.OS === 'ios') {
    data = appStore.versionData.ios
  } else {
    data = appStore.versionData.android
  }
  if (data) {
    const { version, min_version } = data
    if (version) {
      const isNewsetVersion = compareVersion(appStore.currentVersion, version)
      if (isNewsetVersion) {
        if (min_version) {
          const isLowest = compareVersion(appStore.currentVersion, min_version)
          if (isLowest) {
            showVersionAlert(data)
          }
        } else {
          showVersionAlert(data)
        }
      }
    }
  }
}

/* 比较版本
  return
  {
    true   有新版本
    false
  }
*/
const compareVersion = (currentVersion, newestVersion) => {
  // Major.Minor.Patch
  var currentFloat = parseFloat(currentVersion)
  var newestFloat = parseFloat(newestVersion)
  var currentPatchVersion = currentVersion.replace(currentFloat + '.', '')
  var newestPatchVersion = newestVersion.replace(newestFloat + '.', '')
  if (newestFloat > currentFloat) {
    // 线上版本的 Major版本 和 Minor 版本较高
    return true
  } else if (newestFloat < currentFloat) {
    // 线上版本的 Major版本 和 Minor 版本较低
    return false
  } else {
    // 线上版本的 Major版本 和 Minor 版本相等
    if (newestPatchVersion > currentPatchVersion) {
      // 线上版本的 增量版本 较高 有BUG 修复
      return true
    } else {
      return false
    }
  }
}

const fetchNewVersion = () => {
  GET(SERVICE_TYPES.common.FETCH_CHECK_VERSION, {}, success => {
    if (success) {
      const { appStore } = Stories
      appStore.versionData = success
    }
  })
}

const showVersionAlert = newVersion => {
  const { modalStore, appStore } = Stories
  let isShow = true
  if (appStore.latestCheckVersionTime) {
    let days = dateFns.differenceInDays(
      new Date(),
      appStore.latestCheckVersionTime
    )
    if (days < 2) {
      isShow = false
    }
  }
  if (isShow) {
    if (!modalStore.modalVisible) {
      modalStore.show(
        <OccasionBanner activityName={'VersionAlert'} extraData={newVersion} />
      )
      appStore.latestCheckVersionTime = new Date()
    }
  }
}

export { getNewestVersionALert, fetchNewVersion }
