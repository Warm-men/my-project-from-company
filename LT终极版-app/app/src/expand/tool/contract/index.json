{
  "newSubscriptionType": {
    "openQuestions": [
      {
        "title": "1.开通免密支付可以享受哪些特权？",
        "description": [
          {
            "text": "1.享受续费会员每月直减100元特权，该优惠可与优惠券和奖励金叠加"
          },
          {
            "text": "2.享受自在选服务特权，每箱增加2个衣位，将试衣间搬回家"
          },
          { "text": "3.享受「快速发货」特权，下单后新衣箱立即发出" }
        ]
      },
      {
        "title": "2.什么情况下会扣款？",
        "description": [
          {
            "text": "1.在会员到期前第3天上午8点进行续费扣款并延长一个月会员期"
          },
          { "text": "2.在信用账户如产生费用后，将会按相应金额进行扣款" }
        ]
      },
      {
        "title": "3.如何取消免密支付？",
        "description": [
          {
            "text": "除手中有超过一个衣箱的情况外，可至「我的-免密管理」页面随时取消「免密支付」功能"
          }
        ]
      }
    ],
    "contractActiveWithNoReductionWelfare": [
      [
        {
          "text": "1.享受自在选服务特权，每箱增加2个衣位，将试衣间搬回家"
        }
      ],
      [{ "text": "2.享受「快速发货」特权，下单后新衣箱立即发出" }]
    ],
    "contractActiveWelfare": [
      [
        {
          "text": "1.享受续费会员每月直减100元特权，该优惠可与优惠券和奖励金叠加"
        }
      ],
      [{ "text": "2.享受自在选服务特权，每箱增加2个衣位，将试衣间搬回家" }],
      [{ "text": "3.享受「快速发货」特权，下单后新衣箱立即发出" }]
    ],
    "openContractWelfare": {
      "bannerUrl": "https://static.letote.cn/free_password/decrease_banner.png",
      "contentArray": [
        {
          "logoUrl": "https://static.letote.cn/free_password/renewal_offer.png",
          "title": "续费优惠",
          "description": "享受续费会员每月直减100元，该优惠可与优惠券和奖励金叠加"
        },
        {
          "logoUrl": "https://static.letote.cn/free_password/new_auto_pay_time.png",
          "title": "续费时间",
          "description": "平台将在会员到期前第3天上午8点自动续费并延长一个月会员期，如有优惠券和奖励金将会自动使用"
        },
        {
          "logoUrl": "https://static.letote.cn/free_password/repayment.png",
          "title": "自在选服务",
          "description": "自在选服务，可免费开通自在选服务，每箱增加2个衣位，将试衣间搬回家,"
        },
        {
          "logoUrl": "https://static.letote.cn/free_password/credit_account.png",
          "title": "信用账户还款",
          "description": "信用账户如产生费用，将会按相应金额进行自动还款"
        },
        {
          "logoUrl": "https://static.letote.cn/free_password/new_fast_shipping.png",
          "title": "衣箱快速发货",
          "description": "下单后新衣箱立即发出（17点前下单当天发出）不必等待上个衣箱顺丰揽件"
        },
        {
          "logoUrl": "https://static.letote.cn/free_password/new_unbind.png",
          "title": "解除方式",
          "description": "如需取消该功能可至「我的-免密管理」页面进行操作"
        }
      ]
    },
    "dialogMessage": [
      { "text": "你将失去" },
      { "text": "「续费会员直减100元」", "increasedBold": true },
      { "text": "和" },
      { "text": "「自在选服务」", "increasedBold": true },
      { "text": "等特权，是否仍要取消？" }
    ],
    "agreement": {
      "title": "莱尔托特免密支付服务协议",
      "url": "https://wechat.letote.cn/agreement_free_password"
    }
  },
  "subscriptionType": {
    "contractActiveWelfare": [
      [
        { "text": "1.拥有" },
        { "text": "「优先发货」", "increasedBold": true },
        { "text": "特权，下单后新衣箱立即发出（17点前下单当天发出）" }
      ],
      [{ "text": "2.方便会员到期后及时续费，以免错过会员福利；" }],
      [{ "text": "3.在平台中确认支付时，免去每次输入密码等烦扰" }]
    ],
    "contractActiveWithNoReductionWelfare": [
      [
        { "text": "1.拥有" },
        { "text": "「优先发货」", "increasedBold": true },
        { "text": "特权，下单后新衣箱立即发出（17点前下单当天发出）" }
      ],
      [{ "text": "2.方便会员到期后及时续费，以免错过会员福利；" }],
      [{ "text": "3.在平台中确认支付时，免去每次输入密码等烦扰" }]
    ],
    "openQuestions": [
      {
        "title": "1.开通免密支付可以享受哪些特权？",
        "description": [
          { "text": "1.拥有「优先发货」特权；" },
          { "text": "2.方便在会员到期后及时续费，以免错过会员福利；" },
          { "text": "3.在平台中确认支付时，免去每次输入密码等烦扰" }
        ]
      },
      {
        "title": "2.什么情况下会扣款？",
        "description": [
          {
            "text": "会员到期后于次日中午12:00自动扣除下月会费，如有优惠券奖励金自动抵扣",
            "subText": "*季卡及年卡会员到期自动扣除下月会费"
          }
        ]
      },
      {
        "title": "3.如何取消免密支付？",
        "description": [
          {
            "text": "除手中有超过一个衣箱的情况外，可至「我的-免密管理」页面随时取消「免密支付」功能"
          }
        ]
      }
    ],
    "openContractWelfare": {
      "bannerUrl": "https://static.letote.cn/free_password/priority_in_delivery.png",
      "contentArray": [
        {
          "logoUrl": "https://static.letote.cn/free_password/new_fast_shipping.png",
          "title": "衣箱快速发货",
          "description": "下单后新衣箱立即发出（17点前下单当天发出）不必等待上个衣箱顺丰揽件"
        },
        {
          "logoUrl": "https://static.letote.cn/free_password/new_auto_pay_time.png",
          "title": "自动续费时间",
          "description": "会员到期后于次日中午12:00自动扣除下月会费，如有优惠券奖励金自动抵扣",
          "subDescription": "* 季卡及年卡会员到期自动扣除下月会费"
        },
        {
          "logoUrl": "https://static.letote.cn/free_password/new_unbind.png",
          "title": "解除方式",
          "description": "如需取消该功能可至「我的-免密管理」页面进行操作"
        }
      ]
    },
    "dialogMessage": [
      { "text": "取消免密支付，你将失去" },
      { "text": "「优先发货」", "increasedBold": true },
      { "text": "特权，是否仍取消" }
    ],
    "agreement": {
      "title": "莱尔托特免密支付服务协议",
      "url": "https://wechat.letote.cn/agreement_free_password"
    }
  }
}
