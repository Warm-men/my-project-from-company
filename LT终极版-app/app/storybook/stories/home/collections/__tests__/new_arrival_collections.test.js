import React from 'react'
import renderer from 'react-test-renderer'
import ProductItem, * as product_item from '../product_item'
import NewArrivalCollections from '../new_arrival_collections'
import { shallow } from 'enzyme'
import { Column } from '../../../../../src/expand/tool/add_to_closet_status'

product_item.default = jest.fn().mockImplementation(() => {
  return {
    render: () => <div>MockComponent</div>
  }
})

describe('test NewArrivalCollections', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallow(
      <NewArrivalCollections title={'test'} subTitle={'test'} products={[]} />
    )
  })
  describe('test render correctly', () => {
    it('should render correctly when products is []', () => {
      const tree = renderer
        .create(
          <NewArrivalCollections
            title={'test'}
            subTitle={'test'}
            products={[]}
          />
        )
        .toJSON()

      expect(tree).toMatchSnapshot()
    })
  })

  describe('test ProductItem', () => {
    let products
    beforeAll(() => {
      products = Array.from(new Array(9), (val, index) => {
        return { id: index }
      })
    })
    it('should set ProductItem with right input', () => {
      wrapper.setProps({ products })

      const pi = wrapper.find(ProductItem).at(0)
      expect(pi.props().column).toEqual(Column.NewArrivalCollection)
      expect(pi.props().index).toEqual(0)
      expect(pi.props().data).toEqual({ id: 0 })
    })

    it('should call report Data', () => {
      wrapper.setProps({
        products,
        navigation: { state: { routeName: 'test' } }
      })

      const pi = wrapper.find(ProductItem).at(0)
      const result = pi.props().getReportData(0)

      expect(result.column).toEqual(Column.NewArrivalCollection)
      expect(result.router).toEqual('test')
      expect(result.index).toEqual(0)
    })
  })
})
