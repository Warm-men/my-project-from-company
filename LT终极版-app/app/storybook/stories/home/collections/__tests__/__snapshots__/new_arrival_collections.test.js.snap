// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test NewArrivalCollections test render correctly should render correctly when products is [] 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "justifyContent": "center",
          "marginBottom": 22,
          "marginTop": 48,
        },
        undefined,
      ]
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "center",
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "#eee",
            "height": 1,
            "width": 27,
          }
        }
      />
      <Text
        style={
          Object {
            "color": "#000",
            "fontSize": 16,
            "fontWeight": "100",
            "letterSpacing": 1,
            "marginHorizontal": 10,
          }
        }
      >
        test
      </Text>
      <View
        style={
          Object {
            "backgroundColor": "#eee",
            "height": 1,
            "width": 27,
          }
        }
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "marginTop": 8,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#999",
            "fontSize": 8,
            "fontWeight": "500",
            "letterSpacing": 5,
          }
        }
      >
        test
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "flexWrap": "wrap",
        "paddingLeft": 16,
      }
    }
  />
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "opacity": 1,
      }
    }
    testID="test-show-more"
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "center",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#999",
            "fontSize": 12,
            "fontWeight": "400",
            "letterSpacing": 5,
          }
        }
      >
        MORE
      </Text>
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": undefined,
              "fontSize": 18,
            },
            Object {
              "color": "#999",
              "marginLeft": 10,
              "marginTop": 4,
            },
            Object {
              "fontFamily": "Ionicons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        ïŠ„
      </Text>
    </View>
  </View>
</View>
`;
